src/sql/workbench/api/node/extHostModelViewDialog.ts

```diff
@@ -52,7 +52,26 @@ class ModelViewPanelImpl implements sqlops.window.modelviewdialog.ModelViewPanel
 	}
 }
 
-class DialogImpl extends ModelViewPanelImpl implements sqlops.window.modelviewdialog.Dialog  {
+class ViewModelEditorImpl extends ModelViewPanelImpl implements sqlops.workspace.ModelViewEditor {
+	private _content: string;
+	constructor(
+		extHostModelView: ExtHostModelViewShape,
+		private _proxy: MainThreadModelViewDialogShape
+	) {
+		super('modelViewEditor', extHostModelView);
+	}
+
+	public setModelViewId(value: string) {
```

Comment:
I know we do it in the other implementations but I don't think having a private `_content` attribute here that mirrors the private `_modelViewId` attribute from `ModelViewPanelImpl` is the greatest design. I'd suggest just making `_modelViewId` protected on `ModelViewPanelImpl` so that you can access it here

Reply:
Thanks for taking care of this. No need to have the overriden `setModelViewId` method here now since it's not doing anything different

Reply:
Don't see this addressed